src/error.rs

```diff
@@ -198,6 +198,12 @@ pub struct HLLErrors {
     errors: Vec<HLLErrorItem>,
 }
 
+impl Default for HLLErrors {
```

Comment:
It seems weird to implement unused code just to please Clippy.

Reply:
I'm fine with dropping this patch. When clippy is enabled in the CI you will need to explicitly disable that lint.